datastore/zmq/zmq.go

```diff
@@ -71,13 +73,19 @@ func (p *ZMQProducer) Produce(rec *telemetry.Record) {
 	}
 	if nBytes, err := p.sock.SendMessage(telemetry.BuildTopicName(p.namespace, rec.TxType), rec.Payload()); err != nil {
 		metricsRegistry.errorCount.Inc(map[string]string{"record_type": rec.TxType})
-		p.logger.ErrorLog("zmq_dispatch_error", err, nil)
+		p.ReportError("zmq_dispatch_error", err, nil)
 	} else {
 		metricsRegistry.byteTotal.Add(int64(nBytes), map[string]string{"record_type": rec.TxType})
 		metricsRegistry.publishCount.Inc(map[string]string{"record_type": rec.TxType})
 	}
 }
 
+// ReportError to airbrake and logger
+func (p *ZMQProducer) ReportError(message string, err error, logInfo logrus.LogInfo) {
+	p.airbrakeHandler.ReportLogMessage(logrus.ERROR, message, err, logInfo)
+	p.ReportError(message, err, logInfo)
```

Comment:
Infinite recursion. Did you mean `p.logger.ErrorLog`?

Reply:
Same for each of the datastores.

Reply:
hmm.. my branch got messed on rebase. Fixed